frontend/src/reducers/viewHandler.js

```diff
@@ -163,7 +160,7 @@ export default function viewHandler(state = initialState, action) {
           firstRow,
           headerProperties,
           pageLength,
-          result,
+          // result,
```

Comment:
Not sure I understand why this is being removed. Pls explain

Reply:
because we have rows in the tables reducer.